pytest_tests/tests/failovers/test_failover_storage.py

```diff
@@ -56,7 +56,7 @@ def test_storage_node_failover(
             if hard_restart:
                 node_to_stop.kill()
             else:
-                node_to_stop.stop()
+                node_to_stop.stop(force=False)
```

Comment:
But if it fails, we're stuck here, right? Maybe it's better to pass/check some return code when node is killed.

Reply:
no, if it fails - we will get a TimeoutError after 60 seconds

Reply:
flag is about what to do when we get this TimeoutError - with force - we use kill, without force - we raise an error